presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedAggregationsWithExchangeMaterialization.java

```diff
@@ -34,6 +35,16 @@ public void testMaterializedExchangesEnabled()
         assertQuery(getSession(), "SELECT orderkey, COUNT(*) lines FROM lineitem GROUP BY orderkey", assertRemoteMaterializedExchangesCount(1));
     }
 
+    @Test
+    public void testTableWriterMergeEnabled()
```

Comment:
I added this just to verify that the merge operator is there. Since I'm going to enabled it by default - I agree that this test doesn't make much sense then.